src/components/file-upload/file-upload.ts

```diff
@@ -193,7 +194,10 @@ export class MFileUpload extends ModulVue {
     private onClose(): void {
         this.propOpen = false;
         this.$emit('close');
-        this.onCancelClick();
+        this.allFiles
```

Comment:
C'est quoi le bug ici?

PCq c'est étrange de remplacer onCancelClick `
this.propOpen = false;
        this.$refs.dialog.closeDialog();
        this.allFiles
            .filter(f => f.status === MFileStatus.UPLOADING)
            .forEach(this.onUploadCancel);
        this.$emit('cancel');
        this.$file.clear(this.storeName);

`

Par... une partie de l'implémentation de onCancelClick